devices/common_patches/fdcan/fdcan_g4.yaml

```diff
@@ -301,3 +301,11 @@ _include:
     _modify:
       CFIE:
         bitWidth: 3
+
+EXTI:
+  _modify:
+    _interrupts:
+      fdcan1_intr0_it:
+        value: 21
+      fdcan1_intr1_it:
+        value: 22
```

Comment:
Like this:
```suggestion
      fdcan1_intr0_it:
        name: FDCAN1_INTR0_IT
        value: 21
      fdcan1_intr1_it:
        name: FDCAN1_INTR1_IT
        value: 22
```

Reply:
They aren't uppercase in the original .svd file. It doesn't match properly otherwise.